osi_sensorviewconfiguration.proto

```diff
@@ -497,99 +452,73 @@ message LidarSensorViewConfiguration
     //
-    optional uint32 number_of_rays_horizontal = 6;
+    repeated SpatialSignalGain emitted_beam = 5;
 
-    // Number of rays to cast across vertical field of view.
+    // Time offset for each emitted signal.
     //
-    // \note This is a characteristic of the ray tracing engine of the
-    // environment simulation, not a direct characteristic of the sensor.
+    // The time offset in microseconds of every emitted signal
+    // from each frame timestamp.
     //
-    // \rules
-    // is_greater_than_or_equal_to: 1
-    // \endrules
+    // Unit: s
     //
-    optional uint32 number_of_rays_vertical = 7;
+    // \note Usually, this is needed when ray tracing or similar techniques are applied.
+    //
+    // The time_offset_emission_size() must be the same as the emitted_signal_size().
+    //
+    repeated uint32 time_offset_emission = 6;
 
-    // Maximum number of interactions to take into account.
+    // Horizontal beam divergence
     //
-    // \note This is a characteristic of the ray tracing engine of the
-    // environment simulation, not a direct characteristic of the sensor.
+    // The beam divergence in horizontal direction.
     //
-    // \rules
-    // is_greater_than_or_equal_to: 1
-    // \endrules
+    // Unit: rad
     //
-    optional uint32 max_number_of_interactions = 8;
+    optional double beam_divergence_horizontal = 7;
 
-    // Emitter Frequency.
+    // Vertical beam divergence
     //
-    // This information can be used by a ray tracing engine to calculate
-    // doppler shift information and take into account differences in
-    // refraction and reflection. For doppler shift calculations the
-    // sensor model can of course always provide a nominal frequency and
-    // adjust the resulting doppler shift information to actual frequency
-    // through frequency adjustments. For material and geometry interaction
-    // purposes the frequency is also relevant.
+    // The beam divergence in vertical direction.
     //
-    // Unit: Hz
-    //
-    // \rules
-    // is_greater_than_or_equal_to: 0
-    // \endrules
+    // Unit: rad
     //
-    optional double emitter_frequency = 9;
+    optional double beam_divergence_vertical = 8;
 
-    // Number of pixels in frame.
+    // Emitter pose.
     //
-    // This field includes the number of pixels in each frame
+    // Pose of the emitted signal in the sensor coordinate system.
     //
-    // \rules
-    // is_greater_than_or_equal_to: 1
-    // \endrules
-    //
-    optional uint32 num_of_pixels = 10;
+    optional MountingPosition emitter_pose = 9;
 
-    // Ray tracing data.
+    // Receiver pose.
     //
-    // The directions unit vectors describing the Lidar's raster transmission
-    // directions. Length is num_of_pixels \note data is in Lidar's coordinate
-    // system
+    // Pose of the receiver unit in the sensor coordinate system.
     //
-    repeated Vector3d directions = 11;
+    optional MountingPosition receiver_pose = 10;
```

Comment:
Good suggestion. I also added als explanation in transmitter_position ```// In case of lidar sensors this could be e.g. a mirror position.``` and for receiver_position ```// In case of lidar sensors this could be e.g. a photodiode.```